week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,3 +1,25 @@
+function setAlarm() {
+
+    var timmer = document.getElementById("timeRemaining")
+    var counter = document.getElementById("alarmSet").value
+    console.log(counter);
+    timmer.innerHTML = "00:" + counter
+
+
+    setInterval(() => {
+        if (counter > 0) {
+            counter--
+            timmer.innerHTML = "00:" + counter
+            if (counter == 0) {
+                playAlarm()
+            }
+        }
+    }, 1000);
+}
+
+
+ 
+
 function setAlarm() {}
```

Comment:
You've written all the right code, but because there are two definitions of the `setAlarm` function, it doesn't actually get called :) Can you fix that up?